content/docs/command-reference/freeze.md

```diff
@@ -44,51 +44,45 @@ First, let's create a simple DVC-file:
 ```dvc
 $ echo foo > foo
 $ dvc add foo
-Saving information ...
-
-$ dvc run -d foo -o bar cp foo bar
-Running command:
-  cp foo bar
-...
+$ dvc run -d foo -o bar -n make_copy cp foo bar
 ```
 
-Then, let's change the file `foo` that the stage described in `bar.dvc` depends
-on:
+Then, let's change the file `foo` that the stage `make_copy` depends on:
 
 ```dvc
 $ rm foo
 $ echo foo1 > foo
 $ dvc status
 
-bar.dvc
-        deps
-                changed:  foo
-foo.dvc
-        outs
-                changed:  foo
+make_copy:
+	changed deps:
+		modified:           foo
+foo.dvc:
+	changed outs:
+		modified:           foo
 ```
 
-Now, let's freeze the `bar` stage:
+Now, let's freeze the `make_copy` stage:
 
 ```dvc
-$ dvc freeze bar.dvc
+$ dvc freeze make_copy
 $ dvc status
 
-  foo.dvc
-          outs
-                  changed:  foo
+foo.dvc:
+	changed outs:
+		modified:           foo
 ```
 
 Run `dvc unfreeze` to unfreeze it back:
 
 ```dvc
-$ dvc unfreeze bar.dvc
+$ dvc unfreeze make_copy
 $ dvc status
 
-  bar.dvc
-          deps
-                  changed:  foo
-  foo.dvc
-          outs
-                  changed:  foo
+make_copy:
+	changed deps:
+		modified:           foo
+foo.dvc:
+	changed outs:
+		modified:           foo
```

Comment:
Maybe replace this whole chunk (kind of repetitive) with a closing paragraph instead. Something like "This will bring the status of the pipeline back to normal."